lib/oli_web/router.ex

```diff
@@ -379,7 +379,6 @@ defmodule OliWeb.Router do
 
     # Editors
     get("/:project_id/resource/:revision_slug", ResourceController, :edit)
-    get("/:project_id/resource/:revision_slug/activity/:activity_slug", ActivityController, :edit)
```

Comment:
Back in commit 52e906613add34ab859c1e35dff40fe9a8d350aa it looks like the js app this route rendered was removed, cleaning up this route now.